Makefile

```diff
@@ -79,13 +79,27 @@ integrations: build
 test:
 	for PKG in $(PACKAGES); do go test -cover -coverprofile $$GOPATH/src/$$PKG/coverage.out $$PKG || exit 1; done;
 
+.PHONY: test-sqlite
+test-sqlite: integrations.test integrations/gitea-integration
+	GITEA_CONF=integrations/sqlite.ini ./integrations.test
+
 .PHONY: test-mysql
-test-mysql:
-	@echo "Not integrated yet!"
+test-mysql: integrations.test integrations/gitea-integration
+	echo "CREATE DATABASE IF NOT EXISTS testgitea" | mysql -u root
+	GITEA_CONF=integrations/mysql.ini ./integrations.test
 
 .PHONY: test-pgsql
-test-pgsql:
-	@echo "Not integrated yet!"
+test-pgsql: integrations.test integrations/gitea-integration
+	GITEA_CONF=integrations/pgsql.ini ./integrations.test
+
+integrations.test: $(SOURCES)
+	go test -c code.gitea.io/gitea/integrations -tags 'sqlite'
+
+integrations/gitea-integration:
+	curl -L https://github.com/ethantkoenig/gitea-integration/archive/v2.tar.gz > integrations/gitea-integration.tar.gz
```

Comment:
It contains the git repositories for the integration tests. It is necessary, because some integration may need to run git commands, or access files in the repositories.

See also the "STORING THE INTEGRATION ENVIRONMENT" section of the PR description.